packages/firestore/src/exp/database.ts

```diff
@@ -79,7 +79,10 @@ export const CACHE_SIZE_UNLIMITED = LRU_COLLECTION_DISABLED;
  *
  * Do not call this constructor directly. Instead, use {@link getFirestore}.
  */
-export class FirebaseFirestore extends LiteFirestore {
+export class Firestore extends LiteFirestore {
+  /**
+   * whether it's a firestore or firestore-lite instance
```

Comment:
Same comment here as below regarding caps/backticks.

But does this "database.ts" imply RTDB, and is this comment in the right place if so?